test/functional/taef/tests.cpp

```diff
@@ -40,6 +42,327 @@
 
 using namespace Microsoft::VisualStudio::CppUnitTestFramework;
 
+class DriverService {
+    SC_HANDLE ScmHandle;
+    SC_HANDLE ServiceHandle;
+public:
+    DriverService() :
+        ScmHandle(nullptr),
+        ServiceHandle(nullptr) {
+    }
+    bool Initialize(
```

Comment:
please take a look at the other files for style - e.g., each function is surrounded by blank lines, the return type is on its own line, etc.